tests/phpunit/tests/autoloader-classmap.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * Tests related to the WP_Autoload class.
+ *
+ * @group basic
+ */
+class Tests_Autoloader_Classmap extends WP_UnitTestCase {
+
+	/**
+	 * Test that all classes in the classmap are lowercase.
```

Comment:
```suggestion
	 * Tests that all classes in the classmap are lowercase.
```
Test docblocks should also use third-person singular verbs.